Add ChatOps

exAspArk opened the pull request:
<!-- Please fill in the **bold** fields and tick all applicable boxes by placing an "x" inside "[ ]". -->

**[exAspArk/awesome-chatops](https://github.com/exAspArk/awesome-chatops)**

`awesome-chatops` is a collection of awesome bots for ChatOps. ChatOps is a collaboration model that helps to connect people, process, tools, and automation into a transparent workflow (conversation-driven development). 

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

davisonio commented:
> has a Table of Contents section named Contents as the first section

Not done, the 'messengers' section would be good as a sub-category of the main contents page rather than the whole thing.

I think it would be good if this list had more ChatOps related resources (not just bots). e.g. https://www.reddit.com/r/chatops/, community etc. and the items in the sub reddit sidebar. Is that something you would consider?

sindresorhus commented:
Going to the list I had no idea what ChatOps were, having read the list, I still have no idea.

> ChatOps is a collaboration model that helps to connect people, process, tools, and automation into a transparent workflow (conversation-driven development).

This is fine and well, but should also explain why I should care. What makes ChatOps awesome. The first ChatOps word should also link to a resource about it.

-

The list is full of adapters for various messengers, but how are those related to ChatOps.

-

And like @davisonio mentioned, needs more non-library resources.

exAspArk commented:
Hi, thanks a lot for your **awesome** feedback and suggestions! 

* Added the `Contents` section and more resources (videos, books, events, etc.)
* Added a link for the first `ChatOps` word and described why ChatOps is awesome 😸 
* Added a description for `Bots` section
* Added the repository description

sindresorhus commented:
- `Other Resources` is would be better written as `Miscellaneous`

-

> This includes starting each link description with an **uppercase character** and ending it with a period ., and consistent naming

Not followed.

sindresorhus commented on readme.md:
```diff
@@ -440,6 +440,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [DTrace](https://github.com/xen0l/awesome-dtrace) - Dynamic tracing framework.
 - [Userscripts](https://github.com/brunocvcunha/awesome-userscripts) - Enhance your browsing experience.
 - [Pokémon](https://github.com/tobiasbueschel/awesome-pokemon) - Pokémon and Pokémon GO resources and tools.
+- [ChatOps](https://github.com/exAspArk/awesome-chatops) - Bots for ChatOps.
```
Needs a clearer description.

exAspArk replied:
Thanks! Fixed

sindresorhus replied:
No, I mean, the description is just repeating the title (ChatOps). The description should succinctly say what ChatOps is.

exAspArk replied:
@sindresorhus oh, I see.

I've changed the description by using the short phrase of Jesse Newland of GitHub:
> ChatOps – Putting tools in the middle of the conversation.

Is it ok?

sindresorhus replied:
Still pretty unclear what it actually is. I would go for something like:

> Conversation-driven development.

exAspArk replied:
I personally believe that ChatOps can be applied not only for development (also support, marketing, etc.)

What do you think about:

> Managing technical and business operations through a chat.

?

exAspArk commented:
Thanks! 
Updated 👍
